Tests/JSSpecCompiler/test-runner.cpp

```diff
@@ -59,11 +59,11 @@ const Array regression_tests = {
     }
 };
 
-static const LexicalPath path_to_compiler_binary = [] {
+static LexicalPath const path_to_compiler_binary = [] {
```

Comment:
huh I see why east const applies here, but it feels somewhat weird with the `static`

Reply:
east-const-ness has nothing to do with static-ness